Source/AliveLibAO/Abe.cpp

```diff
@@ -9535,9 +9535,197 @@ void Abe::State_155_ElumJumpToFall_42E060()
     // Empty
 }
 
+struct Path_Reset : public Path_TLV
+{
+    __int16 field_18_clearIds;
+    __int16 field_1A_from;
+    __int16 field_1C_to;
+    __int16 field_1E_exclude;
+    __int16 field_20_clearObjects;
+    __int16 field_22_path;
+};
+ALIVE_ASSERT_SIZEOF_ALWAYS(Path_Reset, 0x24);
+
 void Abe::State_156_DoorEnter_42D370()
 {
-    NOT_IMPLEMENTED();
+    switch (field_110_state.door)
+    {
+        case DoorStates::eUnknown_0:
+        {
+            if (field_10_anim.field_4_flags.Get(AnimFlags::eBit18_IsLastFrame))
+            {
+                field_110_state.door = DoorStates::eUnknown_2;
+                field_10_anim.field_4_flags.Clear(AnimFlags::eBit3_Render);
+                field_118 = gnFrameCount_507670 + 3;
+            }
+            return;
+        }
+        case DoorStates::eUnknown_1:
+        {
+            if (field_158_pDeathFadeout->field_6E_bDone)
+            {
+                field_110_state.door = DoorStates::eUnknown_2;
+                field_118 = gnFrameCount_507670 + 5;
+            }
+            return;
+        }
+        case DoorStates::eUnknown_2:
+        {
+            if (field_118 <= static_cast<int>(gnFrameCount_507670))
+            {
+                field_110_state.door = DoorStates::eUnknown_3;
+                field_118 = gnFrameCount_507670 + 3;
+            }
+            return;
+        }
+        case DoorStates::eUnknown_3:
+        {
+            if (field_118 <= static_cast<int>(gnFrameCount_507670))
+            {
+                field_110_state.door = DoorStates::eUnknown_4;
+                auto pTlv = static_cast<Path_Reset*>(gMap_507BA8.TLV_Get_At_446260(
```

Comment:
assign to field_F0_pTlv  missing?

Reply:
not this time, no. At least I can't see it in IDA as well